backend/src/data/migrations/20220207071214_edit_permissions_table.ts

```diff
@@ -0,0 +1,26 @@
+import { Knex } from 'knex';
+import { Permissions, TableName } from '~/common/enums/enums';
```

Comment:
```suggestion
import { TableName } from '~/common/enums/enums';
```

it's danger to use source code in the migrations. create all thing that you need directly in the migration